preview/Dockerfile

```diff
@@ -1,28 +1,34 @@
-FROM buildpack-deps:trusty-scm
+FROM buildpack-deps:jessie-scm
```

Comment:
@naamunds +1 - you can also just install curl with --no-depends and not use `wget`